clients/include/testing_geam_strided_batched.hpp

```diff
@@ -0,0 +1,593 @@
+    {
+        // ROCBLAS
+
+        CHECK_ROCBLAS_ERROR(rocblas_set_pointer_mode(handle, rocblas_pointer_mode_host));
+        CHECK_ROCBLAS_ERROR(rocblas_geam_strided_batched<T>(handle,
+                                                            transA,
+                                                            transB,
+                                                            M,
+                                                            N,
+                                                            &alpha,
+                                                            dA,
+                                                            lda,
+                                                            stride_a,
+                                                            &beta,
+                                                            dB,
+                                                            ldb,
+                                                            stride_b,
+                                                            dC,
+                                                            ldc,
+                                                            stride_c,
+                                                            batch_count));
+
+        CHECK_HIP_ERROR(hC_1.transfer_from(dC));
+
+        rocblas_init<T>(hC_2);
+        CHECK_HIP_ERROR(dC.transfer_from(hC_2));
+
+        CHECK_ROCBLAS_ERROR(rocblas_set_pointer_mode(handle, rocblas_pointer_mode_device));
+
+        CHECK_ROCBLAS_ERROR(rocblas_geam_strided_batched<T>(handle,
+                                                            transA,
+                                                            transB,
+                                                            M,
+                                                            N,
+                                                            d_alpha,
+                                                            dA,
+                                                            lda,
+                                                            stride_a,
+                                                            d_beta,
+                                                            dB,
+                                                            ldb,
+                                                            stride_b,
+                                                            dC,
+                                                            ldc,
+                                                            stride_c,
+                                                            batch_count));
+
+        CHECK_HIP_ERROR(hC_2.transfer_from(dC));
+
+        // reference calculation for golden result
+        cpu_time_used = get_time_us();
+
+        for(int i3 = 0; i3 < batch_count; i3++)
+        {
+            for(int i1 = 0; i1 < M; i1++)
+            {
+                for(int i2 = 0; i2 < N; i2++)
```

Comment:
i2 could be size_t to save you casting to size_t in indexing

Reply:
Thanks,  fixed